app/views/events/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<%if @future.any? || @past.any?%>
```

Comment:
[OPTIONAL]
Consider making this into a helper so that if that condition changes in the future, you wouldn't have to tamper with the view. Plus it helps readability